SQL/0000-00-00-schema.sql

```diff
@@ -117,6 +117,13 @@ DROP TABLE IF EXISTS `mri_scanner`;
 DROP TABLE IF EXISTS `mri_processing_protocol`;
 DROP TABLE IF EXISTS `ImagingFileTypes`;
 
+DROP TABLE IF EXISTS `tarchive_files`;
+DROP TABLE IF EXISTS `tarchive_series`;
+DROP TABLE IF EXISTS `tarchive`;
```

Comment:
why are the DROP TABLE statements for the tarchive tables in this PR?

Reply:
sorry a mistake copy/paste from lines 100-102 above

Reply:
done